en/backup-to-gcs-using-br.md

```diff
@@ -105,6 +105,7 @@ This document provides examples in which the data of the `demo1` TiDB cluster in
     * `spec.br.concurrency`: The number of threads used by each TiKV process during backup. Defaults to `4` for backup and `128` for restore.
     * `spec.br.rateLimit`: The speed limit, in MB/s. If set to `4`, the speed limit is 4 MB/s. The speed limit is not set by default.
     * `spec.br.checksum`: Whether to verify the files after the backup is completed. Defaults to `true`.
+    * `spec.br.timeAgo`: Backs up the data before `timeAgo`. Empty (default) means backing up the current data. [Supports](https://golang.org/pkg/time/#ParseDuration) the data such as "1.5h" and "2h45m".
```

Comment:
ditto

Reply:
Done in https://github.com/pingcap/docs-tidb-operator/pull/721/commits/4917e023f8a6b3c7fa2b683144baeb792bf69a50